net-mgmt/pfSense-pkg-Telegraf/Makefile

```diff
@@ -0,0 +1,41 @@
+# $FreeBSD$
+
+PORTNAME=     pfSense-pkg-Telegraf
+PORTVERSION=  0.0.13
```

Comment:
Any special reason to start a new package with version 0.0.13? Why not simply 0.1?